### PR TITLE
@joeyAghion => Initial stab at MarketInsights

### PR DESCRIPTION
**Pull request by mzikherman**
Tiny bit of progress!

screen shot 2017-11-28 at 12 57 04 pm

### CODE REVIEW

File: src/Components/Artist/MarketInsights.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+export class MarketInsights extends React.Component<MarketInsightsProps, null> {
+  renderGalleryRepresentation() {
+    const { highlights } = this.props.artist
+    const { partner_artists } = highlights
+    if (partner_artists && partner_artists.length > 0) {
+      const partnerArtist = partner_artists[0]
+      const { categories } = partnerArtist.partner
+      let category
+      Object.keys(Categories).forEach(key => {
```
Comment (mzikherman) [inline src/Components/Artist/MarketInsights.tsx]: I can probably write this better, essentially this tries to inspect the embedded partner categories, to figure out which of the 3 special types we have.

File: src/Components/Artist/MarketInsights.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        categories.forEach(partnerCategory => {
+          if (partnerCategory.id === key) {
+            category = Categories[key]
+          }
+        })
+      })
+      return (
+        <div>
+          Represented by {this.props.artist.highlights.partner_artists[0].partner.name}
```
Comment (mzikherman) [inline src/Components/Artist/MarketInsights.tsx]: I should make this more defensive, not rendering if there are no `highlights` or `partner_artists`.

File: src/Components/__stories__/MarketInsights.story.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import { RootQueryRenderer } from "../../Relay/RootQueryRenderer"
+import MarketInsights from "../Artist/MarketInsights"
+
+function ArtistExample(props: { artistID: string }) {
+  return (
+    <RootQueryRenderer
+      query={graphql`
+        query MarketInsightsQuery($artistID: String!) {
+          artist(id: $artistID) {
+            ...MarketInsights_artist @relay(mask: false)
```
Comment (mzikherman) [inline src/Components/__stories__/MarketInsights.story.tsx]: I needed this in order to get the storybook to work...

File: src/Components/Artist/MarketInsights.tsx (hunk 1 above)
Comment (joeyAghion) [reply src/Components/Artist/MarketInsights.tsx]: In practice I think we'll need to apply a hierarchy of preferences, e.g., to say "represented by blue chip galleries..." if there are any in that category, then "...top established..." if there are any in that category, and so on.

**Comment by mzikherman** [comment]
Feature creeped with a 'Top auction Result'

screen shot 2017-11-30 at 4 34 22 pm

File: src/Components/Artist/MarketInsights.tsx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+          partner {
+            name
+            categories {
+              id
+              name
+            }
+          }
+        }
+      }
+      auctionResults(first: 1, sort: PRICE_AND_DATE_DESC) {
```
Comment (mzikherman) [inline src/Components/Artist/MarketInsights.tsx]: aw yea!

**Comment by mzikherman** [comment]
Will close and re-open against `master`, and after some cleanup.

**Pull request closed without merging**